docs/alerts.rst

```diff
@@ -0,0 +1,242 @@
+Alerts
+======
+
+Alerts can be used to notify you about important App Control-monitored activities, such as the
```

Comment:
"can be used" is passive voice. Consider changing the first half of the sentence to "Use alerts to get notifications about important App Control-monitored activities..."

Reply:
Okay, but maybe it should be like this "Use alerts to notify you about important App Control-monitored activities.." because we have actual `notifications` and the user might be confused whether he gets those notifications as objects or he is just getting notified about alerts.